src/main/java/com/synopsys/integration/jenkins/polaris/extensions/buildstep/PolarisBuildStep.java

```diff
@@ -236,7 +129,14 @@ public boolean isApplicable(final Class<? extends AbstractProject> jobType) {
         public String getDisplayName() {
             return DISPLAY_NAME;
         }
-
     }
 
+    private void validateBuild(final AbstractBuild<?, ?> build) throws AbortException {
```

Comment:
I've always been under the impression that all logic a class uses should come before any inner class declarations. Is there a compelling reason to leave validateBuild() after the DescriptorImpl class definition?

Reply:
Sidebar: should validation be moved to the Workflow classes?

Reply:
I agree on both counts. My next commit will move validate() to the Workflow class.